features/providers/check_your_answers.feature

```diff
@@ -146,9 +146,6 @@ Feature: Checking answers backwards and forwards
       Then I should be on a page showing 'Check your answers'
       And the answer for 'Own home' should be 'Yes, with a mortgage or loan'
       And the answer for 'Property value' should be '£500,000.00'
-      And the answer for 'Outstanding mortgage' should be '£100,000.00'
-      And the answer for 'Shared ownership' should be 'Yes, a partner or ex-partner'
-      And the answer for 'Percentage home' should be '50.00%'
```

Comment:
I've removed these three check as the test an assumed previous configuration, rather than the actions specified in feature test. That is, the feature test specifically sets a Property value, but does not touch the values that I've removed.